src/WebJobs.Script/Description/DotNet/FunctionAssemblyLoadContext.cs

```diff
@@ -36,7 +36,7 @@ public FunctionAssemblyLoadContext(string basePath)
                 throw new ArgumentNullException(nameof(basePath));
             }
 
-            _probingPaths.Add(basePath);
+            AddProbingPath(basePath);
```

Comment:
In my testing I noticed that multiple host restarts result in duplicates being added to probing paths - e.g. I had the same bin dir path 5 times in this list

Reply:
👍